subprojects/kotlin-dsl-tooling-builders/src/crossVersionTest/groovy/org/gradle/kotlin/dsl/tooling/builders/r54/KotlinSettingsScriptModelCrossVersionSpec.groovy

```diff
@@ -22,6 +22,7 @@ import org.gradle.test.fixtures.file.LeaksFileHandles
 import org.gradle.util.GradleVersion
 
 import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.not
 
 @TargetGradleVersion(">=5.4")
```

Comment:
Gradle >= 5.4 will not include your changes and behave as you expect.
You should add a new cross version test in `r75` package, duplicated from this one but asserting the new behavior.

Reply:
PTAL, I've separated the test

Reply:
Are you sure about this? I find no example of this. E.g. `KotlinDslScriptsModelCrossVersionSpec` has the package name as `@TargetGradleVersion`, and does not constraint the minimal version.

Is this case here special in any way?